prow/jobs/jobs.yaml

```diff
@@ -6,4 +6,26 @@ periodics:
     serviceAccountName: pre-submit-service-account
     containers:
     - image: alpine
-      command: ["/bin/date"]
\ No newline at end of file
+      command: ["/bin/date"]
+
+postsubmits:
+  #TODO(vijat@): Add support for all service controllers
+  #TODO(vijat@): Find a programmatic way to generate postsubmit jobs for all the service controllers
+  aws-controllers-k8s/apigatewayv2-controller:
```

Comment:
Can we turn this into a preset and re-use it more easily that way?

Reply:
We do not use any properties of podspec supported by presets. So I do not see how we can use presets here.
https://github.com/kubernetes/test-infra/blob/3afb608d28630b99e49e09dd101a96c201268739/prow/config/jobs.go#L35

Presets support Label, Env, Volume and VolumeMount